lmcache/v1/storage_backend/connector/infinistore_adapter.py

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Third Party
+import infinistore
```

Comment:
import the `infinistore` to the `InfinistoreConnector`, so that we can avoid dependent `infinistore` if we do not use infinistore connector at all.